modules/cxf/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,14 +1,14 @@
 module.name=cxf
```

Comment:
This file, should be deleted? Now that CXF is no longer to be a module and instead is part of the compatibility plugin.